integration-testing/election-manager/cypress/integration/yesNoContestTallies.ts

```diff
@@ -0,0 +1,729 @@
+        {
+          '750000017': ['yes'],
+          '750000018': [],
+          '750000015': ['yes'],
+          '750000016': ['no'],
+        },
+        { _precinctId: '6522', _ballotStyleId: '5', _ballotType: 'absentee' }
+      ),
+      generateCVR(
+        electionWithMsEitherNeither,
+        {
+          '750000017': ['no'],
+          '750000018': ['yes'],
+          '750000015': ['yes'],
+          '750000016': ['yes'],
+        },
+        {
+          _precinctId: '6522',
+          _ballotStyleId: '5',
+          _ballotType: 'absentee',
+          _scannerId: 'scanner-2',
+        }
+      ),
+      generateCVR(
+        electionWithMsEitherNeither,
+        {
+          '750000017': ['no'],
+          '750000018': ['yes', 'no'],
+          '750000015': ['no'],
+          '750000016': ['no'],
+        },
+        { _precinctId: '6522', _ballotStyleId: '5', _scannerId: 'scanner-2' }
+      ),
+      generateCVR(
+        electionWithMsEitherNeither,
+        {
+          '750000017': [],
+          '750000018': ['yes'],
+          '750000015': ['no'],
+          '750000016': ['yes'],
+        },
+        { _precinctId: '6522', _ballotStyleId: '5' }
+      ),
+      generateCVR(
+        electionWithMsEitherNeither,
+        {
+          '750000017': ['no'],
+          '750000018': ['no'],
+          '750000015': [],
+          '750000016': ['yes'],
+        },
+        {
+          _precinctId: '6538',
+          _ballotStyleId: '4',
+          _ballotType: 'absentee',
+          _scannerId: 'scanner-2',
+        }
+      ),
+      generateCVR(
+        electionWithMsEitherNeither,
+        {
+          '750000017': ['no'],
+          '750000018': ['no'],
+          '750000015': ['yes', 'no'],
+          '750000016': ['no'],
+        },
+        {
+          _precinctId: '6538',
+          _ballotStyleId: '4',
+          _ballotType: 'absentee',
+          _scannerId: 'scanner-2',
+        }
+      ),
+      generateCVR(
+        electionWithMsEitherNeither,
+        {
+          '750000017': [],
+          '750000018': [],
+          '750000015': ['yes'],
+          '750000016': [],
+        },
+        { _precinctId: '6538', _ballotStyleId: '4' }
+      ),
+      generateCVR(
+        electionWithMsEitherNeither,
+        {
+          '750000017': ['yes', 'no'],
+          '750000018': ['no', 'yes'],
+          '750000015': ['no'],
+          '750000016': ['yes', 'no'],
+        },
+        { _precinctId: '6538', _ballotStyleId: '4' }
+      ),
+    ])
+    cy.visit('/')
+    cy.get('input[type="file"]').attachFile('electionWithMsEitherNeither.json')
+    cy.contains('Election loading')
+    cy.contains('Election Hash: abdfbe6a58')
+    cy.contains('Tally').click()
+    cy.contains('Import CVR Files').click()
+    cy.get('input[data-testid="manual-input"]').attachFile({
+      fileContent: new Blob([fakeCVRFileContents]),
+      fileName: 'cvrFile.jsonl',
+      mimeType: 'application/json',
```

Comment:
If the file is `.jsonl`, then this should probably just be `application/octet-stream` or `text/plain` or something, right? What does the browser actually send when attaching a file like this?